experimental/routes/demo_app_pages.dart

```diff
@@ -13,14 +21,34 @@ class DemoAppPages {
 
   static final routes = [
     GetPage(
-      name: _DemoPaths.demoHome,
-      page: () => const DemoHomeView(),
-      binding: DemoHomeBinding(),
-    ),
-    GetPage(
-      name: _DemoPaths.demoLogin,
+      name: DemoRoutes.demoLogin,
       page: () => DemoLoginView(),
       binding: DemoLoginBinding(),
     ),
+    GetPage(
+      name: DemoRoutes.demoCreate,
+      page: () => const DemoCreateView(),
+      binding: DemoCreateBinding(),
+    ),
+    GetPage(
+      name: DemoRoutes.demoProfile,
+      page: () => const DemoProfileView(),
+      binding: DemoProfileBinding(),
+    ),
+    GetPage(
+      name: DemoRoutes.demoUpdate,
+      page: () => const DemoUpdateView(),
+      binding: DemoUpdateBinding(),
+    ),
+    GetPage(
+      name: DemoRoutes.demoDelete,
+      page: () => const DemoDeleteView(),
+      binding: DemoDeleteBinding(),
+    ),
+    GetPage(
+      name: DemoRoutes.demoHome,
+      page: () => const DemoHomeView(),
+      binding: DemoHomeBinding(),
+    ),
```

Comment:
こっちも全体的に`Profile`の追加お願いします
```suggestion
    GetPage(
      name: DemoRoutes.demoProfileCreate,
      page: () => const DemoProfileCreateView(),
      binding: DemoProfileCreateBinding(),
    ),
    GetPage(
      name: DemoRoutes.demoProfile,
      page: () => const DemoProfileView(),
      binding: DemoProfileBinding(),
    ),
    GetPage(
      name: DemoRoutes.demoProfileUpdate,
      page: () => const DemoProfileUpdateView(),
      binding: DemoProfileUpdateBinding(),
    ),
    GetPage(
      name: DemoRoutes.demoProfileDelete,
      page: () => const DemoProfileDeleteView(),
      binding: DemoProfileDeleteBinding(),
    ),
    GetPage(
      name: DemoRoutes.demoHome,
      page: () => const DemoHomeView(),
      binding: DemoHomeBinding(),
    ),
```

Reply:
view, controller, bindingを上のようにProfileつけたらエラーになると思うので修正お願いします🙏